src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/ancient/tconstruct_melting_pan.json

```
@@ -0,0 +1,16 @@
+{
+  "tool": "tconstruct:melting_pan",
+  "text": [
+    { "text": "熔炼锅能挖掘可熔炼的方块，也可作为手杖使用。可以在地牢、雪屋找到；也可从尸壳，击退袭击后工具匠的礼物中获得。" },
```

```suggestion
    { "text": "熔炼锅能挖掘可熔炼的方块，也可作为手杖使用。可以在地牢、雪屋找到；也可从尸壳身上，击退袭击后工具匠的礼物中获得。" },
```
Polishing, not required.